src/OpenConext/ProfileBundle/Controller/PageController.php

```diff
@@ -20,10 +20,25 @@
 
 use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 
-class DefaultController extends Controller
+class PageController extends Controller
```

Comment:
Please split into separate, meaningful controllers like we discussed IRL. Change route names accordingly.